src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -17,14 +17,14 @@ type BioAndDMSettingProps = {
   onlyReceivedDmFromUserFollowing: boolean;
 };
 
-const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
+const BioAndDMSetting = ({
   bio,
   changeBio,
   avatarUrl,
   following,
   allowAnonDm,
   onlyReceivedDmFromUserFollowing
-}) => {
+}: BioAndDMSettingProps) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `React.FC<BioAndDMSettingProps>` to just passing the props directly in the function signature is a good one. It makes the code more readable and explicit about what props are expected, and it also avoids the implicit `children` prop that comes with `React.FC`. However, you should consider adding return type to the function for better type safety.
```diff
- const BioAndDMSetting = ({
+ const BioAndDMSetting = ({
    bio,
    changeBio,
    avatarUrl,
    following,
    allowAnonDm,
    onlyReceivedDmFromUserFollowing
- }: BioAndDMSettingProps) => {
+ }: BioAndDMSettingProps): JSX.Element => {
   const [isAnonymity, setIsAnonymity] = React.useState(allowAnonDm);
   const [isAllowFollowingSendDM, setIsAllowFollowingSendDM] = React.useState(
     onlyReceivedDmFromUserFollowing
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->